src/components/VmActions/index.js

```diff
@@ -142,9 +200,12 @@ VmActions.propTypes = {
 export default connect(
   (state) => ({
     icons: state.icons,
+    VmAction: state.VmAction,
   }),
   (dispatch, { vm }) => ({
+    checkConsole: () => dispatch(checkConsoleInUse({ vmId: vm.get('id') })),
     onGetConsole: () => dispatch(getConsole({ vmId: vm.get('id') })),
```

Comment:
With this change the overall "Get Console" flow is becoming hard to understand.
To help, please refactor throughout the application getConsole() ---> downloadConsole() and change the action type string accordingly.